extensions/wikia/Email/Controller/WatchedPageController.class.php

```diff
@@ -201,7 +201,7 @@ protected static function getEmailSpecificFormFields() {
 	}
 }
 
-class WatchedPageEditedController extends WatchedPageController {
+class WatchedPageEditedOrCreatedController extends WatchedPageController {
```

Comment:
Just my opinion, but I don't think this more verbose name was necessary. I think, in MediaWiki terms, we already know that a created page is considered an "edit" of that page. I don't care either way though what the class is named.